operators/labInstance-operator/controllers/labinstance_controller.go

```diff
@@ -122,7 +127,7 @@ func (r *LabInstanceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	}
 
 	// prepare variables common to all resources
-	name := "l-" + labTemplate.Name + "-" + fmt.Sprintf("%.8s", uuid.New().String())
+	name := strings.ReplaceAll(labInstance.Name, ".", "-")
```

Comment:
Can we get rid of uuid?

Reply:
I am honestly a bit uncertain about this.

If we consider the labinstances created by the front end, I feel there should be no problem since they are already characterized by a random number (I removed it to avoid having too many sources of randomness). 

On the other hand, if the labinstances are supposed to be created manually, then it may be better to keep it to avoid problems if one is created, removed and then shortly recreated (before the corresponding VM got deleted).

What is your opinion?

Reply:
In the end, I added back the UUID, just to be on the safe side